src/components/Checkout/OrderTotals.vue

```diff
@@ -391,11 +392,11 @@ export default {
 				price: numeral(this.totals.donationTotal)
 			};
 		},
-		isCorporateCampaignPage() {
-			return this.$route.path.substring(0, 4) === '/cc/';
-		}
 	},
 	methods: {
+		ccPageCheck(route) {
+			return isCCPage(route);
```

Comment:
I suggest changing this line to `return isCCPage(this.$route);`, so you can remove the `$route` usage in the templates. My guess is that that could then allow this to be converted to a computed value instead of a method, further simplifying usage to something like `v-if="!ccPageCheck"`

Reply:
Will do! :)

Reply:
Thanks @dyersituations, I would prefer we go with fewer changes as outlined in my comments.